### PR TITLE
Update ingest-data-iot-hub-overview.md

### PR DESCRIPTION
**Pull request by vladikbr**
Update IoTHub connection documentation - add missing property and use correct example.
Depends on https://github.com/MicrosoftDocs/dataexplorer-docs/pull/707

### CODE REVIEW

**Comment by PRMerger14** [comment]
@vladikbr : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/ingest-data-iot-hub-overview.md [hunk 1]
```diff
@@ -72,10 +72,11 @@ IoT Hub exposes the following system properties:
 | iothub-connection-device-id| An ID set by IoT Hub on device-to-cloud messages. It contains the deviceId of the device that sent the message. |
 | iothub-connection-auth-generation-id| An ID set by IoT Hub on device-to-cloud messages. It contains the connectionDeviceGenerationId (as per Device identity properties) of the device that sent the message. |
 | iothub-connection-auth-method| An authentication method set by IoT Hub on device-to-cloud messages. This property contains information about the authentication method used to authenticate the device sending the message. |
+| iothub-creation-time-utc| Timestamp recording when the message was sent by the device. |
 
 If you selected **Event system properties** in the **Data Source** section of the table, you must include the properties in the table schema and mapping.
 
-[!INCLUDE [data-explorer-container-system-properties](includes/data-explorer-container-system-properties.md)]
+[!INCLUDE [data-explorer-container-system-properties](includes/data-explorer-iot-system-properties.md)]
```
Comment (orspod) [inline data-explorer/ingest-data-iot-hub-overview.md:79]:
```suggestion
[!INCLUDE [data-explorer-iot-system-properties](includes/data-explorer-iot-system-properties.md)]
```

**Comment by PRMerger12** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
#sign-off

**Pull request merged**